deploy/gk-deploy

```diff
@@ -388,6 +388,14 @@ else
   exit 1
 fi
 
+if [[ "${CLI}" == "oc" ]]; then
```

Comment:
this should similar to line 392/400 as we save oc=`type oc` which could be /bin/oc